forge/src/main/java/com/gregtechceu/gtceu/blockentity/MetaMachineBlockEntityImpl.java

```diff
@@ -33,41 +33,41 @@ public MetaMachineBlockEntityImpl(BlockEntityType<?> type, BlockPos pos, BlockSt
 
     @Override
     public @NotNull <T> LazyOptional<T> getCapability(@NotNull Capability<T> cap, @Nullable Direction side) {
```

Comment:
This method was quite heavy, should we simplify this in some way?

Reply:
yep I'm considering reworking it, I have some ideas, do you have any suggestions?

Reply:
Ill get a suggestion PR up and running after this one :+1: